### DISCUSSION TITLE
PyScript Next Release Punchlist

### DISCUSSION THREAD

**Discussion by fpliger** [pyscript/pyscript#1653] [General]
Opening this as an umbrella issue to capture all tasks issues and tasks needed to get the first release of PyScript Next out:

* [x] Restore integration tests: This is needed to have the bare minimum level of visibility into what's different (and broken/different/not supported yet/dropped) between the last release of PyScript Classic and PyScript Next
* [x] #1648 - Define a clear understanding of what browsers actually support Next and understand if there's anything we can do to patch in the meantime
* [x] #1651  - Load Python assets as modules, keeping `Globals()` clean and avoiding unintended behaviors
* [x] Restore the Banner to output errors to DOM
* [x] #1691 
* [x] Adapt CDN release automation to new repo structure
* [x] Make `pydom` available
* [ ] Release Changelog
* [x] Documentation
* [x] Release Post: This one is important as this release is a major change for the project and should be well explained and socialized
* [x] Add a clear list of what is supported, what has been deprecated and what will be added back in the next releases

**Comment by WebReflection** [comment]
quickly going through those points:

  - we can't restore integration tests because everything is different ... we can, however, port previous tests (that make sense) to pyscript.core
  - nothing is broken here, in that issue regard, it's an `esm.sh` bug ... see my comments and the issue filed
  - display is not global ... it used to leak at version 0.0.x but everything is reachable as a module as you pointed out yourself with links
  - we can restore outputting errors to DOM but we need to define a better story there ... most users don't want errors in DOM and they recently filed issues about removing that from PyScript
  - agreed on everything else
